src/Traits/ModelTrait.php

```diff
@@ -82,7 +82,18 @@ public function without($tables)
 	 */
 	protected function getWith(): array
 	{
-		return empty($this->with) ? [] : $this->with;
+		// Ensure $this->with is set at all
+		if(empty($this->with)){
+			$this->with = [];
+		}
+		
+		// Force a single table name into an array
+		if (!is_array($this->with))
+		{
+			$this->with = [$this->with];
+		}
+
+		return $this->with;
```

Comment:
Pay attention to styling - we follow CodeIgniter 4's [Style Guide](https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/styleguide.rst).

```suggestion
		// Ensure $this->with is set at all
		if (empty($this->with))
		{
			$this->with = [];
		}
		
		// Force a single table name into an array
		if (! is_array($this->with))
		{
			$this->with = [$this->with];
		}

		return $this->with;
```

Reply:
Thanks. Was it just the first curly bracket or also whitespace?